docs/ml-guide.md

```diff
@@ -355,6 +355,75 @@ jsc.stop();
 {% endhighlight %}
 </div>
 
+<div data-lang="python">
+{% highlight python %}
+from pyspark import SparkContext
+from pyspark.mllib.regression import LabeledPoint
+from pyspark.ml.classification import LogisticRegression
+from pyspark.ml.param import Param, Params
+from pyspark.sql import Row, SQLContext
+
+sc = SparkContext(appName="SimpleParamsExample")
+sqlContext = SQLContext(sc)
+
+# Prepare training data.
+# We use LabeledPoint.
+training = sc.parallelize(
+[LabeledPoint(1.0, [0.0, 1.1,  0.1]),
```

Comment:
Use PEP8 style guidelines for Python: [http://legacy.python.org/dev/peps/pep-0008/]
(The issue here is indentation.)